www/src/components/guidelines/layout.js

```diff
@@ -36,35 +32,17 @@ const Layout = ({ children, background, pathname, pageTitle }) => (
     />
     <Global styles={globalStyles} />
     <Helmet>
-      <title>
-        {pageTitle ? `${pageTitle} | Guidelines | GatsbyJS` : `GatsbyJS`}
-      </title>
-      <meta name="twitter:site" content="@gatsbyjs" />
-      <meta name="og:type" content="website" />
-      <meta name="og:site_name" content="GatsbyJS" />
-      <link rel="canonical" href={`https://gatsbyjs.org${pathname}`} />
-      <html lang="en" />
+      <title>{pageTitle ? `${pageTitle} | Guidelines` : `Guidelines`}</title>
     </Helmet>
-    <Banner />
-    <AnotherHeader pathname={pathname} />
-    <Box
-      bg="background"
-      position="relative"
-      sx={{
```

Comment:
why is `sx` no longer needed?

Reply:
This `sx` pushes content down to account for the banner and the nav-bar. But this is already done by the base `Layout`, meaning that keeping this will cause content to be pushed down even further.

Reply:
👍